.circleci/config.yml

```diff
@@ -126,6 +126,14 @@ workflows:
       - lint:
           filters: *filterAllTags
 
+      - release-branch-forward:
+          requires:
+            - results
```

Comment:
is results the correct trigger here? I see results happens after all the circle tests pass, not after this PR is merged. if i understand correctly, that means this script would run on every passed CI run, not on every merged PR

Reply:
Yes it means that it runs after the results step. That it only applies to the master branch happens here: https://github.com/cri-o/cri-o/pull/3461#discussion_r404253416

Reply:
right, but results happens after every circle test passes (for instance, it's already run and passed on this PR  https://app.circleci.com/jobs/github/openSUSE/cri-o/42837)

this means we are merging from master at the wrong time. It would run every time circle runs and passes, but not every time a PR is merged. I suppose this means that for every PR merged, it would get backported to the last release when the next PR is submitted. I'm not sure that is what you intended here though.

or am I totally off?

Reply:
> right, but results happens after every circle test passes (for instance, it's already run and passed on this PR https://app.circleci.com/jobs/github/openSUSE/cri-o/42837)

The `results` step happens every time, correct. The `requires` only states that `release-branch-forward` runs after the `results` step (from a pipeline perspective).

> this means we are merging from master at the wrong time. It would run every time circle runs and passes, but not every time a PR is merged.

No, the filter `filters branches only master` below let this step run only on the master branch, not on PRs. For example it did not run on this PR, which is correct. But it will run if prow merges the changes into the master. :)

Did I miss anything?

Reply:
@haircommander what do you think?